.github/workflows/k8s-e2e.yaml

```diff
@@ -82,3 +82,21 @@ jobs:
     - run: kubectl apply -f  charts/example/centos-simple-kmod-0.0.1/centos-simple-kmod.yaml
     - run: sleep 360
     - run: kubectl logs `kubectl get pod -n openshift-special-resource-operator | grep special | awk '{ print $1 }'` -c manager -n openshift-special-resource-operator
+  remove:
+    name: remove-image
+    needs:
+      - build
+      - e2e
+    runs-on: ubuntu-latest
+    concurrency:
+      group: ${{ github.workflow }}_e2e-test_$${ github.head_ref }}
+      cancel-in-progress: true
+    steps:
+    - uses: benjlevesque/short-sha@v1.2
+      id: short-sha
+      with:
+        length: 10
+    - run: |
+        curl -s -X GET -H "Authorization: Bearer ${{ secrets.QUAY_API_TOKEN }}" https://quay.io/api/v1/repository/openshift-psap/special-resource-operator/tag/?specificTag=pr-${{ steps.short-sha.outputs.sha }}
+    - run: |
+        curl -s -X DELETE -H "Authorization: Bearer ${{ secrets.QUAY_API_TOKEN }}" https://quay.io/api/v1/repository/openshift-psap/special-resource-operator/tag/pr-${{ steps.short-sha.outputs.sha }}
```

Comment:
Did you check if it is working? I am not sure evaluating GH values like `steps.short-sha.outputs.sha` is working inside `run` commands, I think you may need to add `env:\n\tTAG=${{ steps.short-sha.outputs.sha }}` and then use `TAG` inside the the `run` step

Reply:
It does work, substitutions work without an issue. Besides that, I had to create a fake organization and application for a token, but it does work:
`$ curl -s -X DELETE -H "Authorization: Bearer ***" https://quay.io/api/v1/repository/pacevedo/special-resource-operator/tag/master -i
HTTP/2 204 
date: Mon, 10 Jan 2022 07:54:46 GMT
server: nginx/1.12.1
access-control-allow-origin: *
access-control-allow-methods: PUT, OPTIONS, DELETE
access-control-max-age: 21600
access-control-allow-headers: AUTHORIZATION, CONTENT-TYPE, X-REQUESTED-WITH
x-frame-options: DENY
strict-transport-security: max-age=63072000; preload`

Reply:
OK